packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeader.tsx

```diff
@@ -103,7 +103,12 @@ export const DatepickerCalendarHeader: ComponentWithAs<'div', DatepickerCalendar
         ...unhandledProps,
       })}
     >
-      {createShorthand(Text, label, { defaultProps: () => getA11yProps('label', {}) })}
+      {createShorthand(Text, label, {
+        defaultProps: () =>
+          getA11yProps('label', {
+            className: classes.month,
```

Comment:
Maybe let's call the className `label` to match the slot name
```suggestion
            className: classes.label,
```